docling-server/src/main.py

```
@@ -0,0 +1,108 @@
+
+        # Debug: save result to file
+        debug_path = f"/tmp/debug_result_{doc_id}.json"
+        with open(debug_path, "w") as f:
+            json.dump(result, f, indent=2, default=str)
+        print(f"[DEBUG] Result saved to: {debug_path}")
+
+        return JSONResponse(content=result)
+
+    except HTTPException:
+        raise
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)